dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESMappingAPIImpl.java

```diff
@@ -109,59 +111,43 @@ public ESMappingAPIImpl() {
 		}
 	}
 
-    /**
-     * This method takes a mapping string, a type and puts it as the mapping
-     */
-    public boolean putMapping(final String indexName, final String type, final String mapping)
-            throws ElasticsearchException, IOException {
+	/**
+	 * This method takes a mapping string, a type and puts it as the mapping
+	 * @param indexName
+	 * @param mapping
+	 * @return
+	 * @throws ElasticsearchException
+	 * @throws IOException
+	 */
+	public  boolean putMapping(String indexName, String mapping) throws ElasticsearchException, IOException{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'mapping' is not assigned and could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=3712629)